Documentation/deps-versions.md

```diff
@@ -0,0 +1,9 @@
+|  OCP Version |                                       alertmanager                                       |                                  kubeRbacProxy                                   |                                  kubeStateMetrics                                   |                                                kubernetesMetricsServer                                                 |                                 monitoringPlugin                                  |                                 nodeExporter                                  |                                  promLabelProxy                                  |                                 prometheus                                  |                                    prometheusAdapter                                    |                                  prometheusOperator                                  |                                 thanos                                  |
+|--------------|------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|-------------------------------------------------------------------------------|----------------------------------------------------------------------------------|-----------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| release-4.10 | [0.23.0](https://github.com/openshift/prometheus-alertmanager/blob/release-4.10/VERSION) | [0.11.0](https://github.com/openshift/kube-rbac-proxy/blob/release-4.10/VERSION) | [2.3.0](https://github.com/openshift/kube-state-metrics/blob/release-4.10/VERSION)  | N/A                                                                                                                    | N/A                                                                               | [1.3.1](https://github.com/openshift/node_exporter/blob/release-4.10/VERSION) | [0.4.0](https://github.com/openshift/prom-label-proxy/blob/release-4.10/VERSION) | [2.32.1](https://github.com/openshift/prometheus/blob/release-4.10/VERSION) | [0.9.1](https://github.com/openshift/k8s-prometheus-adapter/blob/release-4.10/VERSION)  | [0.53.1](https://github.com/openshift/prometheus-operator/blob/release-4.10/VERSION) | [0.23.2](https://github.com/openshift/thanos/blob/release-4.10/VERSION) |
```

Comment:
(nit) The link could be to the branch's root directory (e.g. https://github.com/openshift/prometheus-alertmanager/blob/release-4.10).

Reply:
I point it to the file containing the version to ease the retrieval of the commit that upgraded the version.

Reply:
+1 for linking the repo root. For retrieval you could just hardcode the `VERSION` path component, no?

Reply:
done.